src/python/pants/init/target_roots_calculator.py

```diff
@@ -68,3 +142,70 @@ def create(cls, options, build_root=None, change_calculator=None):
       return TargetRoots(tuple(SingleAddress(a.spec_path, a.target_name) for a in changed_addresses))
 
     return TargetRoots(spec_roots)
+
+
+
+class EngineChangeCalculator(object):
+  """A ChangeCalculator variant that uses the v2 engine for source mapping."""
```

Comment:
This comment and the classname are probably both stale: can just take the name `ChangeCalculator` at this point.

Reply:
+1